lib/usage.txt

```diff
@@ -7,6 +7,7 @@ General options:
   --no-color     # Disable colors
   --[no-]insight # Toggle anonymous tracking
   --generators	 # Print available generators
```

Comment:
```suggestion
  --generators   # Print available generators
```

Reply:
Could you help fix this indentation? Looks there is a `\t` behind `--generators`.

Reply:
It looks like this indentation irregularity was preventing tab completion for `--generators` from working at all. The completion tests omitted it, I assume for this reason.

Now that it's fixed I went ahead and added it to the test, as well. :+1: